src/app/shared/components/TopBar/index.tsx

```diff
@@ -1,10 +1,25 @@
-import SelectTeam from './SelectTeam'
+'use client'
+
+import { usePathname } from 'next/navigation'
+
+import BreadCrumb from './BreadCrumb'
+
+const TopBar = () => {
+  const pathname = usePathname()
+
+  const pathArray = pathname.split('/').filter(Boolean)
```

Comment:
interesting, never seen a filter like this

Reply:
Yeh not sure where I came across this but was achieving what I wanted